pootle/apps/pootle_app/project_tree.py

```diff
@@ -49,10 +50,11 @@ def match_template_filename(project, filename):
     :param:`project`.
     """
     name, ext = os.path.splitext(os.path.basename(filename))
+    project_filetypes = ProjectFiletypes(project)
 
     # FIXME: is the test for matching extension redundant?
-    if ext == os.path.extsep + project.get_template_filetype():
-        if ext != os.path.extsep + project.localfiletype:
+    if ext in project_filetypes.template_extensions:
```

Comment:
Sure, I misstyped and it returns a list. But the list is `[".pot"]` while `ext` is `".pot"`.